src/document/DocumentCommandHandlers.js

```diff
@@ -996,6 +1020,7 @@ define(function (require, exports, module) {
                     if (selectedPath) {
                         _doSaveAfterSaveDialog(selectedPath);
                     } else {
+                        checkIfAutoUpdateInProgress();
```

Comment:
Could you explain (workflow) why this call is required here?

Reply:
This is needed for the workflow where user clicked Restart button for the update to happen, but had some dirty files, because of which the app could not be quit, and instead, he got the Unsaved Files dialog, This dialog has 3 buttons : Save/Don't Save/Cancel. If Don't Save is pressed, we quit the app and run the update. If Save is pressed and user saves the file successfully, then also we quit the app and update. But when the user chooses to either press Cancel button in the above dialog, or presses Save followed by Cancel in the Save As dialog, we show him a message saying that the update will now be reflected on relaunch, which inherently means that the update will now be carried out the next time the app quits.
So, for this, we need to know whether or not the save file was cancelled by either of the two scenarios mentioned here, so that we can prompt him with the required message.

Reply:
Thanks for explanation @mbhavi